cfep-23.md

```diff
@@ -0,0 +1,74 @@
+
+## Motivation
+
+Element/Matrix offers a free to use real-time chat platform with web, desktop and mobile clients with end-to-end encryption messaging. Each community or space can create one or more rooms, public or private, to discuss different topics. Other features like videocalls, private messaging, and file sharing are available. Users can create a new account or log in with their Github credentials, among others.
+
+However, it suffers from a few issues that limit its usability within our community. The main concerns are:
+
+- End-to-end encryption fails sometimes, leaving the users unable to read previous messages, or have their messaged read by others. Synchronization of keys across clients is necessary to fix it, but in some other cases the issue has dissapeared spontaneously.
+- Some messages are considered unread for months at a time, and there's no way to find them and mark them as read.
+- The thread system offers very limited ways of tracking conversations and making them discoverable. It's very difficult to follow on topics started hours ago, let alone days or weeks ago.
+- Scoped discussions require a lot of discipline from the users, since threads are optional, and the real-time chat is preferred in the UI.
+
+Zulip, on the other side, is a free to use, open source, real-time discussion board. It has web, desktop and mobile clients. Each community has its Zulip instance, where several "channels" can be created. Each channel hosts a number of threads or topics. Instant messaging is also available within the Zulip instance. Users can also create a new account or log in with their Github credentials, among others.
+
+Zulip offers some features we would consider beneficial to the conda-forge community:
+
+- Threads (or topics) are encouraged and the default user experience.
+- Each topic must have a title that promotes well-scoped discussions, which can be marked as solved when the issue has been addressed.
+- Moderation tools are available to split threads into new ones in case the conversation goes off-topic.
+- Users can "follow" threads they find interesting, which has its own dedicated view. This makes it easier to track updates to conversations you have started or participated in.
+- Private rooms are also available, but they don't offer end-to-end encryption. For this matter, a sensitive communications protocol is proposed below.
+
+## Proposal
+
+After accepting this proposal, we will implement the following decisions:
+
+- The Zulip instance published at `conda-forge.zulipchat.com` will be open to the public with world-readable permissions.
+- The General channel will be the replacement for the `conda-forge/conda-forge.github.io` chat room. This will be reflected in the conda-forge.org documentation, and reminded periodically in the Element room.
+- The conda-forge core team will move its private communications to the Zulip instance, and close the relevant Element rooms.
```

Comment:
We need to leave the old core room open with a note. Many core members are not active and so will loose access otherwise.

Reply:
Also there is a shared conda-forge x bioconda element channel that will have to be kept.

Reply:
An old core member could also reach out via email?

Reply:
Sure but that core room has been around since conda-forge was created. I'm hesitant to simply delete it.

Reply:
Do you have a link for the conda-forge x bioconda room? I can't find it.

Reply:
https://matrix.to/#/#bioconda_core-x-conda-forge:gitter.im